<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>371</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>mainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="browseBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Click the browse button to select a file...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Log</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>781</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>21</width>
      <height>141</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="MusicXMLButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>321</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert to MusicXML</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="MEIButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>321</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert to MEI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CSVButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>321</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert to CSV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MIDIButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>321</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert to CSV to MIDI</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="logBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>781</width>
      <height>121</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="autoFormatting">
     <set>QTextEdit::AutoBulletList</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>browseButton</zorder>
   <zorder>browseBox</zorder>
   <zorder>label</zorder>
   <zorder>line</zorder>
   <zorder>progressBar</zorder>
   <zorder>MusicXMLButton</zorder>
   <zorder>MEIButton</zorder>
   <zorder>CSVButton</zorder>
   <zorder>MIDIButton</zorder>
   <zorder>logBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
